src/implementation/Client/GRPCClient/binding.ts

```diff
@@ -30,7 +30,7 @@ export default class GRPCClientBinding implements IClientBinding {
     const msgService = new InvokeBindingRequest();
     msgService.setName(bindingName);
     msgService.setOperation(operation);
-    msgService.setData(Buffer.from(JSON.stringify(data), "utf-8"));
+    msgService.setData(data instanceof Buffer ? data : Buffer.from(JSON.stringify(data), "utf-8"));
```

Comment:
Please create an abstraction of this.

E.g.

```
import * as SerializerUtil from "./Serializer.util.ts";

msgService.setData(SerializerUtil.serializeGrpc(data))
```

with Serializer.util.ts (see https://github.com/dapr/js-sdk/blob/f1987cb7d3ad77c0943bacb8548971ecaa788a73/src/utils/Error.util.ts for example):

```
export function serializeGrpc(data: any) {
    let serialized = data;

    if (data instanceof Buffer) {
        serialized = data;
    } else {
        serialized = Buffer.from(JSON.stringify(data), "utf-8");
    }

    return serialized;
}
```

+ tests

Reply:
Thanks for your review, I have made the changes according to your suggestion!